docker-hub/api/dvp.yaml

```diff
@@ -0,0 +1,562 @@
+openapi: 3.0.0
+info:
+  title: DVP Data API
+  version: 1.0.0
+  x-logo:
+    url: https://docs.docker.com/images/logo-docker-main.png
+    href: /reference
+  description: |
+    The Docker DVP Data API allows Docker Verified Publishers to view image pull analytics data for their namespaces
+servers:
+  - url: https://hub-stage.docker.com/api/dvp/v1
+security:
+  - HubAuth: []
+
+tags:
+  - name: namespaces
+    x-displayName: Namespace data
+  - name: repos
+    x-displayName: Repository data
+  - name: discovery
+    x-displayName: Discovery
+
+x-tagGroups:
+  - name: API
+    tags:
+      - discovery
+      - namespaces
+      - repos
+
+paths:
+  /data:
+    get:
+      tags: [discovery]
+      summary: Get namespaces and repos
+      description: Gets a list of namespaces and repos which have data available
+      operationId: getNamespaces
+      responses:
+        '200':
+          description: Success
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/NamespaceData'
+  /data/{namespace}:
```

Comment:
I think we were trying to move towards a more standardised url format with a preceding name for each variable

e.g. `/data/namespace/{namespace}`

Whilst more verbose, it allows us to add additional verbs in future such as `/data/users` without the API thinking that `users` is a  namespace. Might be worth asking on #hub-backend to see what the current state of thinking is.

Reply:
I suppose we could do that. I'm not entirely convinced it is necessary in such a simple API - what would `data/users` even refer to?

But I'm happy to make the change.